pkg/gateway/model_build_targetgroup.go

```diff
@@ -375,3 +399,29 @@ func (t *latticeServiceModelBuildTask) buildTargetGroupName(_ context.Context, b
 			t.route.Namespace())
 	}
 }
+
+func buildTargetGroupIpAddressType(_ context.Context, svc *corev1.Service) (string, error) {
+	ipFamilies := svc.Spec.IPFamilies
+
+	isIPv6 := false
+
+	glog.V(6).Infof("buildTargetGroupIpAddressType ipFamilies: %v\n", ipFamilies)
+
+	// Since Lattice Target Group doesn't support dual stack
+	if len(ipFamilies) > 1 {
+		return "", errors.New("Invalid IpFamilies, Lattice Target Group doesn't support dual stack ip addresses")
```

Comment:
looks like it's second place that return dual stack error, if we catch dual stack before, why it's happening here?